github/ci/services/prometheus-stack/hack/deploy.sh

```diff
@@ -11,19 +11,25 @@ main(){
 
     bazelisk run //github/ci/services/prometheus-stack:${environment}.apply
 
+    bazelisk run //github/ci/services/prometheus-stack:${environment}-k8s-services.apply
+
     bazelisk run //github/ci/services/common/k8s/cmd/wait -- -namespace monitoring -selector prometheus-stack-kube-prom-operator -kind deployment
 
     bazelisk run //github/ci/services/common/k8s/cmd/wait -- -namespace monitoring -selector alertmanager-prometheus-stack-kube-prom-alertmanager -kind statefulset
 
-    if [ "${environment}" = "production-control-plane" ] || [ "${environment}" = "testing" ]; then
+    if [ "${environment}" = "production-control-plane" ] || [ "${environment}" = "testing" ] || [ "${environment}" = "production-performance-workloads" ]; then
         bazelisk run //github/ci/services/common/k8s/cmd/wait -- -namespace monitoring -selector grafana -kind deployment
     fi
 
     bazelisk run //github/ci/services/common/k8s/cmd/wait -- -namespace monitoring -selector node-exporter-prometheus-node-exporter -kind daemonset
 
     bazelisk run //github/ci/services/prometheus-stack:${environment}-service-monitors.apply
 
-    bazelisk run //github/ci/services/prometheus-stack:${environment}-rules.apply
+    if [ "${environment}" = "production-control-plane" ] || [ "${environment}" = "testing" ] || [ "${environment}" = "production-e2e-workloads" ]; then
+        bazelisk run //github/ci/services/prometheus-stack:${environment}-common-rules.apply
+    fi
+
+    bazelisk run //github/ci/services/prometheus-stack:${environment}-custom-rules.apply
```

Comment:
Maybe the naming is a bit misleading, common rules are not always applied and custom rules are always applied, shouldn't it be the other way around?

Reply:
I changed the logic, now I am using patches to change the prometheus rules in the performance cluster
So, there is no need for applying different rules